db/migrate/20171114183148_profiles.rb

```diff
@@ -0,0 +1,16 @@
+Sequel.migration do
+
+  up do
+    create_table :profiles do
+      column :id, "uuid", :default=>Sequel::LiteralString.new("uuid_generate_v4()"),
+             :null=>false, :primary_key=>true
```

Comment:
Same rocket operartor thing here

Reply:
I can change, but is it advisable to change an already applied migration script though?  Seems unnecessary to risk changing the behavior

Reply:
Same, please say that this code is already in prod from the start (maybe you said it and I forgot). Not much point to CRing now.